vars/s3up.groovy

```diff
@@ -43,13 +43,18 @@ def call(body) {
         targetObject += config.platform + '/'
     }
     withCredentials([[ $class: 'AmazonWebServicesCredentialsBinding', credentialsId: config.credentialsId]]) {
-        sh """
+        def awscliAvailable = sh(script: 'which aws', returnCode: true) == 0
         echo "Uploading artifacts: ${config}"
-        s3cmd --recursive --no-progres --access_key=${AWS_ACCESS_KEY_ID} --secret_key=${AWS_SECRET_ACCESS_KEY} ${aclPrivate} put ${config.localArtifact} ${targetObject}
-        """
+        if (awscliAvailable) {
+            def aclFlag = config.keepPrivate ? "private" : "public-read"
+            sh "aws s3 cp ${config.localArtifact} ${targetObject} --recursive --no-progres --acl ${aclFlag}"
+        } else {
+            echo yellow("WARNING! Using s3cmd, please upgrade shared lib to test-shared-library@1.7 or install awscli, so faster awscli can be used.")
```

Comment:
Another PR idea: we should have shortcuts like `info`, `debug`, ... which can help to organize Jenkins output.